<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox#sendMsg{
border:none;
}
QGroupBox#onlineUsers{
border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="sendMsg">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="m_lineMsg"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_sendMsg">
        <property name="text">
         <string>信息发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="clickOperators">
     <item>
      <widget class="QTextEdit" name="m_showMsgs"/>
     </item>
     <item>
      <widget class="QListWidget" name="m_friendList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="clickButtons">
       <item>
        <widget class="QPushButton" name="m_deleteFriend">
         <property name="text">
          <string>删除好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_flushFriend">
         <property name="text">
          <string>刷新好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_showOnlineUser">
         <property name="text">
          <string>显示在线用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_findUser">
         <property name="text">
          <string>查找用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_privateChat">
         <property name="text">
          <string>私聊</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="onlineUsers">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QListWidget" name="m_onlineUserList"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_addFriend">
        <property name="text">
         <string>添加好友</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
